src/App.module.css

```diff
@@ -1,3 +1,11 @@
 .App {
   display: flex;
 }
+
+.Translation {
```

Comment:
I have used this class in many places by using composes functionality of modular CSS. Since this class is not used by different types of components so I have put it at the top level in app.css. I am not sure where else can I put it. Should I create a global.css file for this or any other place you can recommend.